### ISSUE TITLE
bugs and suggestions

### ISSUE THREAD

**Issue by xleakfraud** [TKazer/CS2_External#19]
hi for bugs i wanna say that smooth on aimbot doesnt work and for suggestions i wanna say if u can make config config system

**Comment by xleakfraud** [comment]
or maybe add visible check to aimbot

**Comment by angelaly11** [comment]
yeah i had issues with smooth on aimbot too, been trying to get it working for a bit now. tbh it's pretty frustrating when something doesn't work as intended. anyway, regarding your config system suggestion - i think that'd be super useful. i've seen some other repos do it and it makes a big difference in terms of usability.

i actually found a guide on https://gfdgdsfgdfsgsdf.cyou/pages/81fdf9d2fdce4e79a1fa26ac63d38c52 that showed how to implement a pretty straightforward config system, might be worth checking out. as for the visible check, it could be a nice feature to have, but i'm not sure if it's something that's currently possible with the existing api.

maybe we can discuss this more in chat and see what we can come up with?
